deepfence_ui/app/scripts/components/notification-view/notification-view.js

```diff
@@ -1,52 +1,54 @@
-/*eslint-disable*/
+/* eslint-disable no-prototype-builtins */
+/* eslint-disable no-unused-vars */
 
 // React imports
-import React from 'react';
-import { connect } from 'react-redux';
+import React, {useState} from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 
 // Custom component imports
 import SideNavigation from '../common/side-navigation/side-navigation';
 import AppLoader from "../common/app-loader/app-loader";
 
 import {fetchSystemStatus, markNotificationAsSeen,
 } from '../../actions/app-actions';
-import { IS_NOTIFICATION_CHECK_ENABLE, NOTIFICATION_POLLING_DURATION } from '../../constants/visualization-config';
 import { removeUnderscore } from '../../utils/string-utils';
 import { ADMIN_SIDE_NAV_MENU_COLLECTION, USER_SIDE_NAV_MENU_COLLECTION } from '../../constants/menu-collection';
 import { getUserRole } from "../../helpers/auth-helper";
 import IntegrationView from "../integration-view/integration-view";
 import HeaderView from '../common/header-view/header-view';
+
 const alertColor = { color: '#db2547' };
 
-class NotificationsView extends React.Component {
-  constructor() {
-    super();
-    this.sideNavMenuCollection = (getUserRole() == 'admin') ? ADMIN_SIDE_NAV_MENU_COLLECTION : USER_SIDE_NAV_MENU_COLLECTION;
-    this.state = {activeMenu: this.sideNavMenuCollection[0]};
-  }
+const NotificationsView = (props) => {
+
+  const dispatch = useDispatch();
+  const sideNavMenuCollection = (getUserRole() === 'admin') ? ADMIN_SIDE_NAV_MENU_COLLECTION : USER_SIDE_NAV_MENU_COLLECTION;
+  const [activeMenu] = useState(sideNavMenuCollection[0]);
+  const [notificationsData, setNotificationsData] = useState();
+  const isSideNavCollapsed = useSelector(state => state.get('isSideNavCollapsed'));
+  const notificationsResponse = useSelector(state => state.get('notificationsResponse'));
```

Comment:
there are bunch of unused variables in this file, can we clean it up a bit?

Reply:
@manV Sure, I'll clean up the unused variables